examples/BlazorServerApp/Program.cs

```diff
@@ -0,0 +1,51 @@
+using BlazorServerApp.Data;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.FeatureManagement;
+
+namespace BlazorServerApp
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+            builder.Services.AddRazorPages();
+            builder.Services.AddServerSideBlazor();
+            builder.Services.AddSingleton<WeatherForecastService>();
+
+            builder.Services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
+                .AddCookie();
+
+            builder.Services.AddScoped<UserAgentContext>();
+
+            builder.Services.AddScopedFeatureManagement()
+                .WithTargeting<MyTargetingContextAccessor>()
+                .AddFeatureFilter<BrowserFilter>();
+
+            var app = builder.Build();
+
+            // Configure the HTTP request pipeline.
```

Comment:
I think we should remove the comments that aren't relevant to FM. I think they distract from what's important.

Reply:
I guess these comments are from the Blazor server app template. I will remove them.